client/src/components/pages/MembersPage/components/MemberCard.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import styled from 'styled-components'
+
+function Member({ member, deleteMember }) {
+  return (
+    <MemberCard key={member.id + '_visualisation'}>
+      <DeleteMember
+        key={member.id + '_deleteButton'}
+        onClick={() => {
+          deleteMember(member.id.toString())
```

Comment:
Maybe generate ids as strings, so you dont have to call `.toString()` on it all over the place?